src/ReactiveDomain/ReactiveDomain.Tests/Logging/with_event_store.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Threading;
+using EventStore.ClientAPI;
+using ReactiveDomain.Domain;
+using ReactiveDomain.EventStore;
+using ReactiveDomain.Tests.Specifications;
+
+namespace ReactiveDomain.Tests.Logging
+{
+    // ReSharper disable once InconsistentNaming
+    public abstract class with_event_store :
+        CommandBusSpecification,
+        IDisposable
+    {
+        protected EventStoreLoader EventStore;
+        protected IEventStoreConnection TcpEndPointConnection;
+        protected IRepository Repo;
+
+        private static readonly IPEndPoint IntegrationTestTcpEndPoint;
+
+        static with_event_store()
+        {
+            BootStrap.Load();
+            IntegrationTestTcpEndPoint = new IPEndPoint(IPAddress.Loopback, 1113);
+        }
+
+        protected override void Given()
+        {
+
+            TcpEndPointConnection = EventStoreConnection.Create(IntegrationTestTcpEndPoint);
+            EventStore = new EventStoreLoader();
+
+            TcpEndPointConnection.ConnectAsync().Wait();
+
+            //TODO: better path!  Tacky because PerkinElmer\Greylock...
+            EventStore.SetupEventStore(
+                new DirectoryInfo(
+                    @"c:\program files\PerkinElmer\Greylock\eventStore"));
+
+            Repo = new GetEventStoreRepository(EventStore.Connection);
+
+            Thread.Sleep(2000);
+            // is this where I need the sleep, or in child?
```

Comment:
Why do you need a sleep at all?

Reply:
It takes a while for things to get set up the first time. It definitely wasn't ready if I just plowed right onward, but clearly there was a question in my mind about where it actually needs it. I'll look to see if I removed the other sleep, then get rid of the comment - or investigate further if needed. Maybe there is a cleaner "are you ready" detection mechanism.

Reply:
Removed here, left in when_messages_en/disabled